Add missing errors to our API docs and revamp the look of error tables for increased readability (#279)
alphagov/notifications-tech-docs: source/documentation/error_tables/_get_the_status_of_one_message.md

**ValidationError&nbsp;(status&nbsp;code&nbsp;400)**|
`id is not a valid UUID`|Check the notification ID.|
**NoResultFound&nbsp;(status&nbsp;code&nbsp;404)**|

> `No result found`|Check when your message was sent. If it was sent before the retention period, it has been deleted. You can no longer get the status of this message. The default retention period is 7 days.|

Reviewer: ```suggestion
`No result found`|Your message was sent before the retention period and has been deleted. You can no longer get the status of this message. The default retention period is 7 days.|
```

Author: I think the original message works better here. That's because "No result found" could be due to two factors - either the message was sent outside of the retention period, or it never existed in the first place.

The original message tells user to check when the message was sent - whereas the proposition say confidently "it was sent before, and that's why you get no result".

In addition to the above, you may also encounter:

* various schema validation errors, for example when you forget to pass in an argument, or pass in an argument of a wrong type.

> * errors that are not related to getting the status of a message, but instead are related to things like authentication and rate limits. You can find a list of these errors [in the General errors section](#general-errors)

Reviewer: ```suggestion
* errors that are not related to getting the status of a message, but instead are related to things like authentication and rate limits. You can find a list of these errors [in General errors](#general-errors)
```
